lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -0,0 +1,541 @@
+#ifndef TRITON_CONVERSION_TRITONGPU_TO_LLVM_CONVERSION_PATTERN_H_
+#define TRITON_CONVERSION_TRITONGPU_TO_LLVM_CONVERSION_PATTERN_H_
+
+#include "Utility.h"
```

Comment:
We need to include absolute path

Reply:
Here I followed MLIR style, e.g. https://github.com/llvm/llvm-project/blob/main/mlir/lib/Conversion/GPUCommon/GPUOpsLowering.cpp.

The thing is that Utility.h is in lib but not include, because we will not expose them into the users. In my understanding, we only use absolute path for files in include paths.

Reply:
I see, it makes sense.